bin/test-merges.js

```diff
@@ -141,7 +141,15 @@ function checkoutGitCommit(commitId) {
           reject(err2);
           return;
         }
-        resolve(commitId);
+
+        // Pass along the original commit message
+        git.show(["-s", "--format=%B"], (err3, data3) => {
+          if (err3) {
+            reject(err3);
+            return;
+          }
+          resolve([commitId, data3.trim()]);
```

Comment:
Fixed JSDoc. The return isn't allowed with m-c recommended eslint rules: https://hg.mozilla.org/mozilla-central/file/tip/tools/lint/eslint/eslint-plugin-mozilla/lib/configs/recommended.js#l236